plugins/woocommerce/package.json

```diff
@@ -481,25 +481,29 @@
 					}
 				},
 				{
-					"name": "Core API tests - HPOS disabled",
+					"name": "Core API tests (HPOS:off)",
```

Comment:
Changes (Core API tests (HPOS:off)):
- run in PRs
- do not track patterns/templates changes in PRs as those unlikely to impact REST (trunk push will cover us)
- changes in other packages are irrelevant, except one exception